docs/setup/turn/eturnal.md

```diff
@@ -0,0 +1,153 @@
+# eturnal TURN server
+
+The following sections describe how to install [eturnal](<https://github.com/processone/eturnal>) (which implements the TURN REST API).
+
+## `eturnal` setup
+
+### Initial installation
```

Comment:
hypocritical since we briefly discuss how to install coturn in the docs, but wondering if we should just link people to the documentation at https://github.com/processone/eturnal#installation so that we don't need to worry about it falling out of date.

Reply:
Yes, I will run over the configuration part again. I have an improvement in mind. Will commit the improvements later on the weekend.